tensorboard/webapp/notification_center/_views/notification_center_component.ng.html

```diff
@@ -18,6 +18,23 @@
   <mat-icon svgIcon="notifications_none_24px"></mat-icon>
 </button>
 
-<mat-menu #menu="matMenu">
-  <div mat-menu-item>notification 1</div>
+<mat-menu #menu="matMenu" class="notification-menu">
+  <div>
+    <div *ngFor="let notification of notifications" class="menu-item">
+      <div>
+        <mat-icon
+          svgIcon="{{ notification.icon }}"
+          class="category-icon"
+        ></mat-icon>
+        <span>{{ notification.category }}</span>・
```

Comment:
Instead of printing the ENUM value onto the string and make that load bearing, use ngSwitch and hardcode some of the values here. Also make sure you don't render the `span` when the category is not set or NONE.

When done, please omit the string value on the enum.

Reply:
Screen shots for this modification (no icon and no category for the second notifiaiton)
![Screen Shot 2021-03-08 at 5 14 48 PM](https://user-images.githubusercontent.com/1131010/110403361-d62da300-8031-11eb-805d-449e5f0f9488.png)